homeassistant/components/zha/entity.py

```diff
@@ -66,6 +66,8 @@ def name(self) -> str | UndefinedType | None:
         original_name = super().name
 
         if original_name not in (UNDEFINED, None) or meta.fallback_name is None:
+            if meta.postfix is not None:
+                return f"{original_name} {meta.postfix}"
```

Comment:
Matter in HA puts the postfix in brackets. We may wanna do that as well. For reference, see:
https://github.com/home-assistant/core/blob/180f898bfa8b4bc3a1a5a97f64866c64fe1f551e/homeassistant/components/matter/entity.py#L190